src/pipewire.c

```diff
@@ -245,7 +219,10 @@ pipewire_remote_destroy (PipeWireRemote *remote)
       pw_loop_destroy_source (loop, g_steal_pointer (&remote->roundtrip_timeout));
     }
 
+  spa_hook_remove (&remote->registry_listener);
```

Comment:
It's actually possible that the crashes in fbe26534e14c48481d578da6908d47a53915b77b were missing this...